app/javascript/components/modals/sources/component.jsx

```diff
@@ -30,23 +30,27 @@ class ModalSources extends PureComponent {
   }
 
   getContent() {
-    const { data: { title, body } } = this.props;
+    const { data: { body } } = this.props;
 
     return (
       <div className="c-modal-sources">
-        <h3>{title}</h3>
         <div className="body">{this.parseContent(body)}</div>
       </div>
     );
   }
 
   render() {
     const { open, setModalSources, data } = this.props;
+    const lowerCaseTitle = data && data.title && data.title.toLowerCase();
+    const formattedTitle = lowerCaseTitle
```

Comment:
What is this madness ❓ I think we have control over what data is received no? Can we not make the data different so we dont need to do this?